libs/portals/shared-modules/delegations/src/components/access/AccessConfirmModal.tsx

```diff
@@ -54,8 +55,9 @@ export const AccessConfirmModal = ({
     setError(formError)
   }
 
-  const toName = delegation?.to?.name
-  const toNationalId = delegation?.to?.nationalId
+  const toName = (delegation as AuthCustomDelegationOutgoing)?.to?.name
```

Comment:
Can you change the component props to use the right type if this modal is only for outgoing delegations?

Reply:
Good catch